contracts/contracts/chains/polygon_zkevm/PolygonZkBridgeInterface.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+
+interface IPolygonZkEVMBridge {
+    function bridgeMessage(
+        uint32 destinationNetwork,
+        address destinationAddress,
+        bool forceUpdateGlobalExitRoot,
+        bytes calldata metadata
+    ) external payable;
+}
```

Comment:
Have we considered submitting an issue to OpenZeppelin to add this to their `vendor` set?
They seem to already have polygon there:
https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/vendor/polygon

Reply:
https://github.com/OpenZeppelin/openzeppelin-contracts/issues/4247 @istankovic you are free to edit it :D